.rubocop.yml

```diff
@@ -18,6 +18,12 @@ AllCops:
     - 'truffleruby/target/classes/**/*'
     - 'mxbuild/**/*'
 
+Lint/AssignmentInCondition:
+  Description: Don't use assignment in conditions.
+  StyleGuide: "#safe-assignment-in-condition"
+  Enabled: true
+  AllowSafeAssignment: true
+
```

Comment:
And FWIW, I find the assignment in conditionals isn't just error-prone, but also leads to code that's harder to read. I have to make a mental context switch when I encounter it. Having said that, I'm not sure that `()` is enough to alleviate the problem since `()` have other valid usages in a conditional.